### ISSUE TITLE
Suicide burn seems to only take vertical speed into account, not horizontal

### ISSUE THREAD

**Issue by SkyMarshal** [CYBUTEK/KerbalEngineer#39]
I don't have any screen shots at present, but I'll gladly take some as this has bit me in the ass before.

**Comment by space-is-hard** [comment]
Non-vertical suicide burns are much more complex calculations to consider. See [this reddit thread](http://www.reddit.com/r/Kos/comments/3478p2/the_math_and_physics_of_suicide_burns/) for some discussion on the matter.

**Comment by SkyMarshal** [comment]
That's essentially what I thought.  I guess my issue is that it is marked as generic suicide burn information, not explicitly vertical.  I slammed into the Mün twice before I figured it out.  (Not that I mind killing my horizontal velocity before the suicide burn, I've grown accustomed to burning retrograde at a low periapsis beforehand, it just took me a bit to realise that the KER data was mislabeled.)

**Comment by swilbur** [comment]
I've edited my copy of KER to use the following calculation:

1. First, burn directly retrograde (relative to the surface)  to cancel all of your horizontal velocity.
2. At the end of that burn, your vertical velocity will not quite be zero, since gravity has been pulling on you during the burn.  burn due up until your velocity is zero.

With that plan, it's easy to calculate your desired suicide burn height. (much easier than solving the full differential equation) Note that it will always overestimate the desired height, so there's an automatic safety margin built in.

Also notice that (unmodded) KER gives you an incorrect value for horizontal velocity. (it calls the correct function, but KSP returns the wrong number) I've changed that to sqrt((surface speed)^2 - (vertical velocity)^2) in my copy.

**Comment by SkyMarshal** [comment]
That's a fairly reasonable way to do it... and I always wondered about the horizontal velocity part.  I mean, it should change *a little* due to the fact that the planet(oid) is not a flat plane--so the reference "horizontal" plane changes as you move over/around it--but it changed a lot more than that, which was super noticeable when you were doing a near-vertical descent.

**Comment by harryyoung** [comment]
Worked on with https://github.com/CYBUTEK/KerbalEngineer/issues/93 and https://github.com/CYBUTEK/KerbalEngineer/pull/95
